etna/media/blocks.py

```diff
@@ -1,9 +1,21 @@
-from wagtail.core import blocks
+from django.forms.utils import ErrorList
+from django.templatetags.static import static
+from django.utils.html import format_html
+
+from wagtail.core import blocks, hooks
+from wagtail.core.blocks.struct_block import StructBlockValidationError
 from wagtail.images.blocks import ImageChooserBlock
 
 from wagtailmedia.blocks import AbstractMediaChooserBlock
 
 
+@hooks.register("insert_editor_css")
+def editor_css():
+    return format_html(
+        '<link rel="stylesheet" href="{}">', static("/css/dist/etna-wagtail-editor.css")
+    )
+
+
```

Comment:
Wagtail hooks should always be registered in a `wagtail_hooks.py` file in an app where possible, as Wagtail always looks for them in that location. I would suggest moving this code there to make things easier to find for developers coming from other Wagtail projects.

Reply:
Thanks @ababic - that makes sense!